apps/heureka-legacy/package.json

```diff
@@ -50,7 +50,7 @@
     "@cloudoperators/juno-communicator": "*",
     "@cloudoperators/juno-messages-provider": "*",
     "@cloudoperators/juno-ui-components": "*",
-    "@cloudoperators/juno-url-state-provider-v1": "^1.3.2",
+    "@cloudoperators/juno-url-state-provider": "*",
```

Comment:
Do we want to make changes into the heureka-legacy?

Reply:
I think yes, otherwise we'll not be able entirely remove the dependency `juno-url-state-provider-v1`